internal/impl/parquet/processor_encode.go

```diff
@@ -30,19 +32,24 @@ func parquetEncodeProcessorConfig() *service.ConfigSpec {
 		// Stable(). TODO
 		Categories("Parsing").
 		Summary("Encodes https://parquet.apache.org/docs/[Parquet files^] from a batch of structured messages.").
-		Field(parquetSchemaConfig()).
-		Field(service.NewStringEnumField("default_compression",
-			"uncompressed", "snappy", "gzip", "brotli", "zstd", "lz4raw",
+		Fields(
+			parquetSchemaConfig().Optional(),
+			service.NewStringField("schema_metadata").
+				Description("Optionally specify a metadata field containing a schema definition to use for encoding instead of a statically defined schema. For batches of messages the first messages schema will be applied for all subsequent messages of the batch.").
```

Comment:
```suggestion
				Description("Optionally specify a metadata field containing a schema definition to use for encoding instead of a statically defined schema. For batches of messages, the first message's schema will be applied to all subsequent messages of the batch.").
```